src/version.cpp

```diff
@@ -7,12 +7,13 @@
 ////////////////////////////////////////////////////////////////////////////////
 
 #include <hpx/config/defines.hpp>
-#include <hpx/version.hpp>
 
 #if defined(HPX_HAVE_PARCELPORT_MPI)
 #include <mpi.h>
 #endif
 
+#include <hpx/version.hpp>
```

Comment:
If this is yet another order dependent `#include`, please annotate as such to avoid regressions.